src/lib/components/PeriodicTable/PeriodicTable.svelte

```diff
@@ -43,35 +43,12 @@
 	export let selected: any[] = [];
 	export const clear = () => clearSelection();
 
-	const group_names = new Map<string, string>([
-		['H/He', 'period 1'],
-		['Li/Ne', 'period 2'],
-		['Na/Ar', 'period 3'],
-		['K/Kr', 'period 4'],
-		['Rb/Xe', 'period 5'],
-		['Cs/Rn', 'period 6'],
-		['Fr/Lr', 'period 7'],
-		['La/Yb', 'lanthanide'],
-		['Ac/No', 'actinide'],
-		['H/Fr', 'alkali'],
-		['Be/Ra', 'alkaline'],
-		['Sc/Lr', 'group 3'],
-		['Ti/Hf', 'group 4'],
-		['V/Ta', 'group 5'],
-		['Cr/W', 'group 6'],
-		['Mn/Re', 'group 7'],
-		['Fe/Os', 'group 8'],
-		['Co/Ir', 'group 9'],
-		['Ni/Pt', 'group 10'],
-		['Cu/Au', 'group 11'],
-		['Zn/Hg', 'group 12'],
-		['B/Tl', 'triels'],
-		['C/Pb', 'tetrels'],
-		['N/Bi', 'pnictogen'],
-		['O/Po', 'chalcogen'],
-		['F/At', 'halogen'],
-		['He/Rn', 'noble gas'],
-	]);
+	let group_names: any = {};
```

Comment:
@GlistenSTAR TS offers us `Map<string, string>`, please, let's use it.

Reply:
> Please, adopt for `Map`

Done